diff -Naur org.old/apache/html/dom/HTMLFrameElementImpl.java org/apache/html/dom/HTMLFrameElementImpl.java
--- org.old/apache/html/dom/HTMLFrameElementImpl.java	2018-04-21 04:02:12.000000000 +0000
+++ org/apache/html/dom/HTMLFrameElementImpl.java	2020-01-30 00:59:10.137995270 +0000
@@ -16,6 +16,7 @@
  */
 package org.apache.html.dom;
 
+import org.w3c.dom.Document;
 import org.w3c.dom.html.HTMLFrameElement;
 
 /**
@@ -127,6 +128,16 @@
         setAttribute( "src", src );
     }
 
+    public String getContent()
+    {
+        return null;
+    }
+
+    public Document getContentDocument()
+    {
+        return null;
+    }
+
     
     /**
      * Constructor requires owner document.
diff -Naur org.old/apache/html/dom/HTMLIFrameElementImpl.java org/apache/html/dom/HTMLIFrameElementImpl.java
--- org.old/apache/html/dom/HTMLIFrameElementImpl.java	2018-04-21 04:02:12.000000000 +0000
+++ org/apache/html/dom/HTMLIFrameElementImpl.java	2020-01-30 00:59:16.563995700 +0000
@@ -16,6 +16,7 @@
  */
 package org.apache.html.dom;
 
+import org.w3c.dom.Document;
 import org.w3c.dom.html.HTMLIFrameElement;
 
 /**
@@ -150,6 +151,16 @@
         setAttribute( "width", width );
     }
 
+    public String getContent()
+    {
+        return null;
+    }
+
+    public Document getContentDocument()
+    {
+        return null;
+    }
+
     
     /**
      * Constructor requires owner document.
diff -Naur org.old/apache/html/dom/HTMLObjectElementImpl.java org/apache/html/dom/HTMLObjectElementImpl.java
--- org.old/apache/html/dom/HTMLObjectElementImpl.java	2018-04-21 04:02:09.000000000 +0000
+++ org/apache/html/dom/HTMLObjectElementImpl.java	2020-01-30 00:59:01.984994724 +0000
@@ -16,6 +16,7 @@
  */
 package org.apache.html.dom;
 
+import org.w3c.dom.Document;
 import org.w3c.dom.html.HTMLObjectElement;
 
 /**
@@ -239,6 +240,15 @@
         setAttribute( "width", width );
     }
 
+    public String getContent()
+    {
+        return null;
+    }
+
+    public Document getContentDocument()
+    {
+        return null;
+    }
     
     
     /**
diff -Naur org.old/apache/xerces/xpointer/XPointerHandler.java org/apache/xerces/xpointer/XPointerHandler.java
--- org.old/apache/xerces/xpointer/XPointerHandler.java	2018-04-21 04:02:13.000000000 +0000
+++ org/apache/xerces/xpointer/XPointerHandler.java	2020-01-30 00:58:00.146990587 +0000
@@ -1055,7 +1055,7 @@
 
             return;
         }
-        super.startElement(element, attributes, augs);
+        //super.startElement(element, attributes, augs);
     }
 
     /**
@@ -1144,7 +1144,7 @@
             // no need to restore restoreBaseURI() for xml:base and xml:lang processing
             return;
         }
-        super.endElement(element, augs);
+        //super.endElement(element, augs);
     }
 
     /**
@@ -1249,4 +1249,4 @@
         super.setProperty(propertyId, value);
     }
 
-}
\ No newline at end of file
+}
